src/main/antora/modules/ROOT/pages/jdbc/template.adoc

```diff
@@ -0,0 +1,48 @@
+[[jdbc.template]]
+= Template API
+
+Spring Data JDBC offers the javadoc:org.springframework.data.jdbc.core.JdbcAggregateTemplate[] as a more direct means to load and persist entities in a relational database.
+Repositories use to a large extend the `JdbcAggregateTemplate` to implement their features.
+
+This section will highlight only the most interesting parts of the `JdbcAggregateTemplate`.
+For a more complete overview see the JavaDoc of `JdbcAggregateTemplate`.
+
+== Accessing the JdbcAggregateTemplate
+
+`JdbcAggregateTemplate` is intended to be used as Spring bean.
+If you have set up your application including Spring Data JDBC you may configure a dependency on `JdbcAggregateTemplate` in any Spring bean and the Spring Framework will inject a properly configured instance.
```

Comment:
Change to:

If you have set up your application to include Spring Data JDBC, you can configure a dependency on `JdbcAggregateTemplate` in any Spring bean, and the Spring Framework injects a properly configured instance.